### DISCUSSION TITLE
WSJT-X Reference UDP Message Aggregator Server

### DISCUSSION THREAD

**Discussion by Vulp35** [kg4vdk/arcos-linux-modules#36] [It's Not In The Manual...Promise!]
I've been playing around with arcOS a bit the last couple of weeks (**Loving it by the way!**). Today I decided to give WSJT-X Message Aggregator a look and I have no clue what it's purpose is. It's very obviously pulling the activity from the local instance of WSJT-X UDP port. I've tried googling it to maybe get a video overview or webpage for it, but it seems to be a generic enough term I just get a bunch of stuff about the different programs you can have pull in data over UDP for WSJT-X.

Could someone here maybe give me an overview or links to resources?

Thanks and 73,
KF0JOU

**Comment by kg4vdk** [comment]
I, too, was completely unfamiliar with it. But, you intrigued me and I started looking. There does seem to be very little in the way of documentation. This older forum post was the most helpful information I could find. Seems like it could be used to aggregate all the WSJT-X traffic on a local network (e.g. Field Day).

https://www.ahrdf.net/forum/thread-1030.html

Maybe someone else has used it more and can elaborate.

**Reply by Vulp35** [reply]
Yup, that link was the most I was able to find as well! Maybe I will have to get together with some other hams and give it a shot.
